lib/decanter/core.rb

```diff
@@ -102,19 +102,11 @@ def unhandled_keys(args)
             .select { |handler| handler[:type] != :input }
             .map { |handler| "#{handler[:name]}_attributes".to_sym }            
 
-        if unhandled_keys.any?
-          case strict_mode
-          when true
-            p "#{self.name} ignoring unhandled keys: #{unhandled_keys.join(', ')}."
-            {}
-          when :with_exception            
-            raise(UnhandledKeysError, "#{self.name} received unhandled keys: #{unhandled_keys.join(', ')}.")
-          else
-            args.select { |key| unhandled_keys.include? key }
-          end
-        else
-          {}
+        return {} unless unhandled_keys.any?
+        if strict_mode 
```

Comment:
ruby guard syntax? :grin: 

`raise(UnhandledKeysError, "#{self.name} received unhandled keys: #{unhandled_keys.join(', ')}.") if strict_mode `

Reply:
The line is so longgg

But yes I can do that.